kotlin/workflow-testing/src/test/java/com/squareup/workflow/testing/RealRenderTesterTest.kt

```diff
@@ -41,11 +41,18 @@ class RealRenderTesterTest {
   @Test fun `expectWorkflow with output throws when already expecting workflow output`() {
     // Don't need an implementation, the test should fail before even calling render.
     val workflow = Workflow.stateless<Unit, Nothing, Unit> {}
+    @Suppress("UNCHECKED_CAST")
     val tester = workflow.renderTester(Unit)
-        .expectWorkflow(ChildWorkflow::class, rendering = Unit, output = EmittedOutput(Unit))
+        .expectWorkflow(
+            ChildWorkflow::class, rendering = Unit,
+            output = EmittedOutput(Unit) as EmittedOutput<Nothing>
```

Comment:
Casting is strange, `EmittedOutput `with `Nothing` shouldn't be possible. Would it make sense to convert `EmittedOutput` to a sealed class - first implementation contains an actual output value, second implementation is just a singleton representing `Nothing` for emitted output?

Reply:
Like you said, this is not a normal thing to do. Not inclined to change `EmittedOutput` to be able to express nonsense. Frankly, I was just banging on these tests as little as possible to keep them working.

I could take a stab at changing `workflow`'s output type to `String` or `Unit`, but I don't want to sink a lot of time in here.